### DISCUSSION TITLE
Have an "open" research community call once a month

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#381] [Ideas]
- What: Create an open research community call once per month
- Why
  - Situation: we currently have weekly calls for (initial) research (interest) group (following from #261). New people would like to join the community. also some people like Boaz and Jamie aren't able to come to interest group every time.
  - Complication: weekly calls are big commitment for some people. we have potential new joiners who need to find out what it is about.
- When: monthly call on Friday at 1330 European
- Where: online on hangouts
- Who: open to all

Next actions

- [x] Discuss the proposal
- [x] Agree to take forward or now **✅2023-03-25 i think most people seem to be happy so taking this forward**

Assuming goes forward, create issue ...

- [ ] Schedule a new monthly meeting for this e.g. "Research Group Open Call Monthly"
- [ ] Add attendees

**Comment by baouroux** [comment]
Sounds good... Very complementary to what I guess will be the increasing focus of the weekly calls

**Comment by msantolini** [comment]
Yes, very good! We can set on a date tomorrow for this? Like decide whether to do 1st, 2nd, 3rd or 4th Friday, what kind of facilitation to do, etc?

**Reply by rufuspollock** [reply]
Last time i think we already said 2nd week of the month (that is usually good as does not clash with e.g. new year, or end of year etc) and there is stably a 2nd friday in the month!

**Comment by msantolini** [comment]
Great, I guess you meant 2nd _Friday_ of the month?
Then this would mean 14th of April?

**Reply by rufuspollock** [reply]
Yes, meant friday and updated text to say that.

**Comment by rufuspollock** [comment]
Based on discussion and reflection today i think there is some significant useful clarifications:

- This is about establishing an open research *community* call once a month
- This would take place at the same time (and therefore "replace") the regular weekly call of the one currently active research (interest) group

I'm going to update the proposal appropriately and for ease of comparison i'm copying current version of it below:

## Current Version

> - What: Designate one per month of research group calls as open.
> - Why
>   - Situation: we currently have weekly calls for research group #261. New people would like to join. also some people like Boaz and Jamie who don't come every time. Also we have
>   - Complication: weekly calls are big commitment for some people. we have potential new joiners who need to find out what it is about.
> - When: monthly call on Friday at 1330 European
> - Where: online on hangouts
> - Who: open to all

**Comment by rufuspollock** [comment]
Ok, i think we have agreement to move forward and have created follow on issue for this: https://github.com/life-itself/community/issues/388
